doc/man/8/radosgw-admin.rst

```diff
@@ -274,6 +274,10 @@ Options
 
 	Temporary url key.
 
+.. option:: --max-buckets
```

Comment:
Just curious why this option is not placed at end?
Why this is placed in b/w?

Reply:
I wanted to place it just before `--system` as in `radosgw-admin` usage.

But as other options are missing, that's probably not the perfect place, I can either place it a the end, or keep it here and create another PR for other missing options.

Reply:
@ClemPi might want to mention that "max-bucket" is 1000 by default.